template/hapi/server.js

```diff
@@ -18,21 +20,20 @@ server.ext('onPreResponse', function(request, reply) {
         var response = request.response;
         return reply(response.state('config', encodeURIComponent(internals.clientConfig)));
     }
-    else {
-        return reply();
-    }
+
+    return reply.continue();
 });
 
 
 // require moonboots_hapi plugin
-server.pack.register({plugin: require('moonboots_hapi'), options: moonbootsConfig}, function (err) {
+server.register({ register: require('moonboots_hapi').register, options: moonbootsConfig }, function (err) {
```

Comment:
It is in hapi 8: http://hapijs.com/api#serverregisterplugins-options-callback

If you change moonboots_hapi to module.exports the register function it won't be though: https://github.com/wraithgar/moonboots_hapi/blob/master/index.js#L43